Makefile

```diff
@@ -151,6 +146,31 @@ manifests-generate: $(CONTROLLER_GEN)
 manifests-kustomize: $(KUSTOMIZE)
 	$< build config/default > config/render/capm3.yaml
 
+.PHONY: set-manifest-image-bmo
+set-manifest-image-bmo: $(KUSTOMIZE) manifests
+	$(info Updating container image for BMO to use ${MANIFEST_IMG}:${MANIFEST_TAG})
+	cd config/default && $(abspath $(KUSTOMIZE)) edit set image quay.io/metal3-io/baremetal-operator=${MANIFEST_IMG}:${MANIFEST_TAG}
+
+.PHONY: set-manifest-image-ironic
+set-manifest-image-ironic: $(KUSTOMIZE) manifests
+	$(info Updating container image for Ironic to use ${MANIFEST_IMG}:${MANIFEST_TAG})
+	cd ironic-deployment/default && $(abspath $(KUSTOMIZE)) edit set image quay.io/metal3-io/ironic=${MANIFEST_IMG}:${MANIFEST_TAG}
```

Comment:
In CAPM3 we do a global substitution of all `image: *` in the manifest, but this does not work well with Ironic, since we have multiple different containers (e.g. mariadb). For this reason I decided to use `kustomize edit set image` instead so we replace only the mentioned images.

Please also note that I'm reusing the same variables in all targets (`MANIFEST_IMG` and `MANIFEST_TAG`). My thinking is that you cannot run all the targets at the same time so they may as well use the same. If you think it would be better with separate variables (e.g. `MANIFEST_IMG_BMO` and `MANIFEST_IMG_IRONIC`), please let me know.

Reply:
You can run several targets, but I think it's an acceptable simplification.